components/collection/drop/DropConfirmModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <NeoModal
+    :value="isModalActive"
+    :can-cancel="['outside', 'escape']"
+    scroll="clip"
+    class="top"
+    content-class="modal-width"
+    @close="onClose">
+    <ModalBody :title="$t('drops.finalizeClaimNow')" @close="onClose">
+      <div>
+        <ModalIdentityItem />
+      </div>
+
+      <p class="py-5 is-capitalized">
+        {{ $t('drops.subscribe') }}
+      </p>
+
+      <form @submit.prevent="confirm">
+        <NeoInput
+          ref="emailInput"
+          v-model="email"
+          type="email"
+          required
+          :placeholder="$t('mint.nft.email.placeholder')" />
+
+        <div class="pt-5">
+          <NeoCheckbox v-model="agree" class="is-capitalized">
+            {{ $t('drops.consent') }}
+          </NeoCheckbox>
+        </div>
+
+        <div class="is-flex is-justify-content-space-between pt-4">
+          <NeoButton
+            class="is-flex is-flex-1 btn-height is-capitalized shine"
```

Comment:
replace `btn-height`

```suggestion
            class="is-flex is-flex-1 h-14 is-capitalized shine"
```